papers/orhan_eroglu/00_geocat.rst

```diff
@@ -190,23 +188,23 @@ usage examples.
    :cite:`geocatcontribguide` :label:`fig3structure`
 
 Many of the computational functions in GeoCAT are implemented in pure Python.
-However, there are some others that are implemented in Fortran but wrapped up
-in Python. To facilitate contribution, the whole GeoCAT-comp structure is split
-into two repositories with respect to being either pure-Python or Python with
-compiled codes (i.e. Fortran) implementations. Such implementation layers are
-handled within GeoCAT-comp and GeoCAT-f2py repositories, respectively.
-
-GeoCAT-comp code-base does not explicitly contain or require any compiled
-code, making it more accessible to the general Python community at large.
-In addition, GeoCAT-f2py is automatically installed through GeoCAT-comp
+However, there are others that remain implemented in Fortran, but are wrapped
+in Python. The whole GeoCAT-comp structure is split
+into two repositories with respect to being either pure Python or Python with
+compiled code (i.e. Fortran) implementations. Such implementation layers are
+handled with the GeoCAT-comp and GeoCAT-f2py repositories respectively.
```

Comment:
Should we mention that we are trying to phase out GeoCAT-f2py?

Reply:
Even though we are trying to implement pure Python substitutes for f2py functions, we will likely not phase out f2py. That's because there is still much compiled code in unstructured grid workflows that would be contributed into geocat-comp stack.